partcad/tests/unit/test_part_enrich.py

```
@@ -29,21 +29,32 @@ def test_part_enrich_get_1():
-    assert brick.config["parameters"]["width"]["default"] == 20.0
+    assert cube_enrich.config["parameters"]["width"]["default"] == 20.0
 
 
-# TODO(clairbee): add support for the following one
-# def test_part_enrich_get_3():
-#     """Load a CadQuery part using enrichment for parameters and see if the parameters changed"""
-#     ctx = pc.Context("examples/produce_part_cadquery_primitive")
-#     brick = ctx._get_part(":brick2")
-#     assert brick is not None
-#     assert asyncio.run(brick.get_wrapped()) is not None
+def test_part_enrich_get_3():
```

Ruff (0.8.2)

Missing return type annotation for public function `test_part_enrich_get_3`

Add return type annotation: `None`

(ANN201)